crates/codegen/runtime/generator/src/ir/model.rs

```diff
@@ -430,12 +434,14 @@ impl IrModelBuilder {
             }
         };
         let has_nonterminals = fields.iter().any(|field| !field.is_terminal);
+        let multiple_operators = expression.operators.len() > 1;
```

Comment:
I think I'm missing something: Do we require/enforce that all `PrecedenceOperator` siblings have the same field names/shapes?

- If so, I wonder if the issue can still happen with a single operator, if it points to something non-unique (like an enum non-terminal). Should we determine if an operator is lided or not, based on its uniqueness rather than number of operators?
- If not, would that require using enums for each operator?